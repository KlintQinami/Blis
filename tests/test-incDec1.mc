int main() {
  int x = 0;
  int y = 2;
  int z = 1336;
  int w = 43;
  printi(x++);
  printi(y--);
  printi(x);
  printi(y);
  printi(++z);
  printi(--w);
  return 0;
}
