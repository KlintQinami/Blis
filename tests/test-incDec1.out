0
2
1
1
1337
42
